simulation.py

```diff
@@ -0,0 +1,79 @@
+"""
+Fuctions for a process to simulate an alpha.
+"""
+
+import copy
+import importlib
+import sys
+from thousandaire.constants import DATA_LIST_ALL
+from thousandaire.constants import TRADING_CONFIGS, TRADING_REGIONS
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+PRICE_DATASET = 'price_dataset'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays_all = {
+        region : raw_data['workdays'][region]
+        for region in TRADING_REGIONS if region in raw_data['workdays']}
+    return_data = {}
+    for region, workdays in workdays_all.items():
+        return_data[region] = copy.deepcopy(raw_data)
+        for dataset in return_data[region].values():
+            dataset.set_workdays(workdays)
+    return_data['workdays'] = workdays_all
+    return return_data
+
+def extract_data(raw_data, data_list, price_dataset, region):
+    """
+    Extract the data which alpha need.
+
+    Because the raw_data is combined with all data we have, we need to pure
+    the data ,which the alpha need, to pass for saving resource.
```

Comment:
I think this is redundant. This function can be used in other ways -- if any. These comments should be put in `simulate` because that's how it uses this function.

Reply:
Instead, please describe the output format of this function in the comment.